sdk/armada-protocol-common/src/simulator/interfaces/IArmadaSimulatedPosition.ts

```diff
@@ -0,0 +1,41 @@
+import { z } from 'zod'
+import { ArmadaPositionDataSchema, IArmadaPosition } from '../../common/interfaces/IArmadaPosition'
+
+/**
+ * @interface IArmadaSimulatedPosition
+ * @description Interface for an Armada Protocol simulated position, used in the simulator only
+ */
+export interface IArmadaSimulatedPosition extends IArmadaPosition, IArmadaSimulatedPositionData {
```

Comment:
I need more context why it's needed. let's catch up

Reply:
It is for now, but the idea is to have a custom position with `deposit` and `withdraw` methods that update the current deposited balance in the position. This comes from the DMA simulator in which Damian added custom functions to track balances, where it would have been better to have a `LendingSimulatedPosition` with those functions as methods that would simulate those operations. I can expand in a call if needed

Reply:
Looks good thanks